fpdf/table.py

```diff
@@ -268,23 +280,124 @@ def _render_table_cell(
                 if style
                 else FontFace(fill_color=self._cell_fill_color)
             )
-        with self._fpdf.use_font_face(style):
-            page_break, cell_height = self._fpdf.multi_cell(
-                w=col_width,
-                h=row_height,
-                txt=cell.text,
-                max_line_height=self._line_height,
-                border=self.get_cell_border(i, j),
-                align=text_align,
-                new_x="RIGHT",
-                new_y="TOP",
-                fill=fill,
-                markdown=self._markdown,
-                output=MethodReturnValue.PAGE_BREAK | MethodReturnValue.HEIGHT,
-                wrapmode=self._wrapmode,
-                **kwargs,
+
+        if cell.padding:
+            from fpdf.fpdf import get_padding_tuple
+
+            padding = get_padding_tuple(cell.padding)
+        else:
+            padding = self._padding
+
+        if cell.v_align:
+            v_align = cell.v_align
+        else:
+            v_align = self._v_align
+
+        # place cursor (required for images after images)
+        cell_widhts = [self._get_col_width(i, jj) for jj in range(j)]
```

Comment:
Its seems that `colspan` is ignored there

Reply:
agree, will test

Reply:
How is colspan expected to work exactly?

Should I put in an empty cell for the cell that is being covered by the cell spanning multiple columns?

Reply:
Documented how colspan is working in #807 

Colspan works fine in combination with padding:
<img width="566" alt="image" src="https://github.com/PyFPDF/fpdf2/assets/34062862/f01bef1d-a538-424a-a819-534c73be5e7d">

but fails in combination with gutter, but that is a pre-existing bug (#808 )